src/jit/codegenarm.cpp

```diff
@@ -504,8 +558,15 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
             assert(varTypeIsIntegralOrI(treeNode));
             __fallthrough;
 
+        case GT_ADD_LO:
+        case GT_ADD_HI:
+        case GT_SUB_LO:
+        case GT_SUB_HI:
         case GT_ADD:
         case GT_SUB:
+            genConsumeOperands(treeNode->AsOp());
+            genCodeForBinary(treeNode);
+            break;
```

Comment:
nit: please add a blank line after the `break;`

---
In reply to: [105069635](https://github.com/dotnet/coreclr/pull/9978#discussion_r105069635) [](ancestors = 105069635)